Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/Conversation.kt

```diff
@@ -108,7 +113,11 @@ fun ConversationContent(
                             Message(authorMe, content, timeNow)
                         )
                     },
-                    scrollState = scrollState,
+                    resetScroll = {
+                        scope.launch {
```

Comment:
Nit: add comment about `smootScrollToItem()`

Reply:
This one is ok as a snap because it's triggered by user input